_mappings/supported-field-types/join.md

```diff
@@ -15,21 +15,32 @@ redirect_from:
 **Introduced 1.0**
 {: .label .label-purple }
 
-A join field type establishes a parent/child relationship between documents in the same index. 
+A `join` field type defines a parent/child relationship between documents within the same index. It is a field that records how documents are related so that queries can connect related documents:
```

Comment:
```suggestion
A `join` field type defines a parent/child relationship between documents within the same index. It performs the following to record how documents are related so that queries can connect related documents:
```